.vib/jaeger/cypress/cypress/integration/jaeger_spec.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+it('list and retrieve jaeger traces', () => {
+
+    const testService = 'redis';
+    const currentDate = new Date(); 
+    const timestampMillis = currentDate. getTime() * 1000;
+    // Check application availability though the UI
+    cy.visit('/search?end=' + timestampMillis + '&limit=20&lookback=1h&maxDuration&minDuration&service=' + testService + '&start=0').then(() => {
+
+        // Ensure page contains Traces in an H2 tag
+        cy.contains('h2', 'Traces');
+
+        let traceIDLink = '';
+        // Get <a> tag link of class "ResultItemTitle--item ub-flex-auto"
+        cy.get('a.ResultItemTitle--item.ub-flex-auto').then((href) => {
```

Comment:
As you are interested in the `href` value, you may chain the Cypress command `invoke()` to obtain it:

```suggestion
        cy.get('a.ResultItemTitle--item.ub-flex-auto').invoke('attr', 'href').then((href) => {
```

With that, you avoid the creation of the var `traceIDLink` in line 14.

Reply:
Done [27e4031](https://github.com/bitnami/charts/pull/13480/commits/27e403182ebebe1e024001baf5a0a5d49e856582)